### DISCUSSION TITLE
Solution for mysterious hidden ssid in both bands (TG799Xtream)

### DISCUSSION THREAD

**Discussion by boytunp** [hack-technicolor/hack-technicolor#90] [General]
Hi, 

I recently rooted my Telenor vant-w 18.1 TG799Xtream router by reading the related issue like 100 times. I think it would be super nice if someone added the Telenor  type 2 firmware to the wiki. 

Anyway now I see these mysterious hidden ssid broadcasts definetly broadcasted by my router on both bands. I tried to enable disable guest ssids but it didn't help. 

I vaguely remember reading somewhere that ISPs might use wireless meshing to provide wifi anywhere to their customers stealing other customers' bandwidth .

If somebody can confirm this is a feature of the router and can be disabled in the firmware please show me how. And of course  it would be nice add it to the wiki. 

Thanks
**Describe Feature Request**

**Additional context**

**Comment by LuKePicci** [comment]
> I recently rooted my Telenor vant-w 18.1 TG799Xtream router by reading the related issue like 100 times. I think it would be super nice if someone added the Telenor  type 2 firmware to the wiki.

They were included already since #87 

> I see these mysterious hidden ssid broadcasts definetly broadcasted by my router on both bands.

Have a look into /etc/config/wireless

**Comment by boytunp** [comment]
I meant readthedocs page. There is no section for Telenor and the section for telia does not list the telenor firmware. Maybe I misunderstood the association between this github and the readthedocs page.
https://hack-technicolor.readthedocs.io/en/stable/Repository/#tg799vac-xtream-vant-w

I investigated the /etc/config/wireless 

With my limited experience I could not see anything that might cause this. Only listings are related to my two APs (5 and 2.4Ghz) and the guest APs which are disabled as in the example
```
config wifi-iface 'wl1_1'                                                             
        option device 'radio_5G'                                                      
        option mode 'ap'                                                              
        option vlan_id '3'                                                            
        option network 'wlnet_b_5'                                                    
        option ssid 'Guest_5GHz_XXXXX'                                               
        option reliable_multicast '0'                                                 
        option state '0'  

```

**Comment by LuKePicci** [comment]
Every git branch matches a wiki version, by default you see the stable version (stable branch), but latest version (master branch) is also online. The README on the bottom of the github repo homepage explains the mappings https://github.com/kevdagoat/hack-technicolor You will see Telenor in stable branch when that branch is mature.

A wireless hotspot having hidden SSID would still be part of those listed in there. Maybe we need some more hints about what and how you discovered those hidden interfaces.

**Comment by boytunp** [comment]
Thank you förr the information.
